plugins/eos/gs-plugin-eos.c

```diff
@@ -2185,20 +2185,15 @@ should_add_os_upgrade (GsApp *os_upgrade)
 
 	return FALSE;
 }
-
-gboolean
-gs_plugin_refresh (GsPlugin *plugin,
-		   guint cache_age,
-		   GsPluginRefreshFlags flags,
-		   GCancellable *cancellable,
-		   GError **error)
+static void
```

Comment:
Nitpick: Missing blank line before this function.

Reply:
Fixed this